modules/appeals_api/spec/requests/v1/notice_of_disagreements_controller_spec.rb

```diff
@@ -216,7 +216,7 @@ def base_path(path)
       uuid = create(:notice_of_disagreement).id
       get("#{path}#{uuid}")
       expect(response.status).to eq(200)
-      expect(parsed.dig('data', 'attributes', 'formData')).to be_a Hash
+      expect(parsed.dig('data', 'attributes', 'formData')).to be_nil
```

Comment:
We should't be returning the key at all, since it's never going to be present. I think there are ways that you can set the serializer to do this automatically.

Reply:
The formData key is not included, but you are correct, the test would pass if an empty form_data hash were returned.
I changed these checks to confirm that there is no form-data key returned